### PR TITLE
fix(cli): Fix sync in protov3

### PR DESCRIPTION
**Pull request by yevgenypats**
This requires additional update in plugin-pb-go. For delete stale we actually don't want the schema but just the name of the table

### CODE REVIEW

File: cli/cmd/sync_v3.go [hunk 1]
```diff
@@ -249,3 +242,28 @@ func syncConnectionV3(ctx context.Context, sourceClient *managedplugin.Client, d
 	fmt.Printf("%s. Resources: %d, Errors: %d, Warnings: %d, Time: %s\n", msg, totalResources, totals.Errors, totals.Warnings, syncTimeTook.Truncate(time.Second).String())
 	return nil
 }
+
+func tableNameFromSchema(sc *arrow.Schema) string {
+	tableName, _ := sc.Metadata().GetValue("cq:table_name")
```
Comment (disq) [inline cli/cmd/sync_v3.go:247]: Is there a const we can use here? Maybe move `cq:*` consts to `plugin-pb-go` if we don't want to import the SDK?

Comment (yevgenypats) [reply cli/cmd/sync_v3.go:247]: yeah we can move this later to `plugin-pb-go` just didn't want to do it now.

File: cli/cmd/sync_v3.go [hunk 2]
```diff
@@ -249,3 +242,28 @@ func syncConnectionV3(ctx context.Context, sourceClient *managedplugin.Client, d
...
+	return tableName
+}
+
+func deleteStale(client plugin.Plugin_WriteClient, tables map[string]bool, sourceName string, syncTime time.Time) error {
+	for tableName := range tables {
+		// TODO remove this print
+		fmt.Println("Deleting stale records from table", tableName)
```
Comment (hermanschaaf) [inline cli/cmd/sync_v3.go:254]: Should we remove the print now?

File: cli/cmd/sync_v3.go [hunk 3]
```diff
@@ -249,3 +242,28 @@ func syncConnectionV3(ctx context.Context, sourceClient *managedplugin.Client, d
...
+		if err := client.Send(&plugin.Write_Request{
+			Message: &plugin.Write_Request_Delete{
+				Delete: &plugin.Write_MessageDeleteStale{
+					SourceName: sourceName,
+					SyncTime:   timestamppb.New(syncTime),
+					// TODO change table
```
Comment (hermanschaaf) [inline cli/cmd/sync_v3.go:260]: We still need to  add the table name here, right?

File: cli/cmd/sync_v3.go (hunk 2 above)
Comment (yevgenypats) [reply cli/cmd/sync_v3.go:254]: yeah going to remove once proto-buf PR is merged

File: cli/cmd/sync_v3.go [hunk 4]
```diff
@@ -249,3 +242,26 @@ func syncConnectionV3(ctx context.Context, sourceClient *managedplugin.Client, d
...
+}
+
+func deleteStale(client plugin.Plugin_WriteClient, tables map[string]bool, sourceName string, syncTime time.Time) error {
+	for tableName := range tables {
+		if err := client.Send(&plugin.Write_Request{
+			Message: &plugin.Write_Request_Delete{
+				Delete: &plugin.Write_MessageDeleteStale{
+					SourceName: sourceName,
+					SyncTime:   timestamppb.New(syncTime),
+					TableName: tableName,
```
Comment (hermanschaaf) [inline cli/cmd/sync_v3.go:258]: It looks like this file is not gofmt-ed

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**